### ISSUE TITLE
scroll inside the item 

### ISSUE THREAD

**Issue by anshusharma13** [yuyakaido/CardStackView#97]
i need vertical scroll , animation stopped while putting scroll inside the item ,please let me know it anybody having the idea about this.

**Comment by luongvo** [comment]
@anshusharma13 check this https://github.com/yuyakaido/CardStackView/pull/76

**Comment by anshusharma13** [comment]
Thank you  for the response but I need scroll in the particular card

**Comment by luongvo** [comment]
@anshusharma13 yes, my PR here is just the way to customize the card touch events, you must go ahead with your customization to handle both scroll view and card swiping

**Comment by luongvo** [comment]
@anshusharma13 this is an example from my solution to enable both vertical viewpager scrolling and card swiping https://github.com/Buddify/CardStackView/tree/support_card_container_view_customizable_viewpager_example

**Comment by anshusharma13** [comment]
hello @luongvo this example is not working, I am stuck in this from last 1 month, please if you have some other things then please let me know .

**Comment by anshusharma13** [comment]
Thank you so much @luongvo its working but while vertical scrolling its stuck, have you any idea about this?

**Comment by anshusharma13** [comment]
I think its done while we stop vertical swiping?

**Comment by anshusharma13** [comment]
@luongvo  vertical scrolling stop when we scroll till the end, please let me know if you have any idea?

**Comment by luongvo** [comment]
@anshusharma13 try to adjust touch events by your self there

**Comment by Rammohandubey1990** [comment]
Have you goten solution for scroll vertically in cardstackview ?

**Comment by Rammohandubey1990** [comment]
not yet
